dev/integration_tests/module_host_with_custom_build_v2_embedding/app/build.gradle

```diff
@@ -14,7 +14,7 @@ android {
 
     defaultConfig {
         applicationId "io.flutter.addtoapp"
-        minSdkVersion 19
+        minSdkVersion 21
```

Comment:
Note to self: this can't be `flutter.minSdkVersion` because this is an "add to app" style test.